### PR TITLE
feat(web): added root path to agent script

### PR DESCRIPTION
**Pull request by robot9706**
Added an input to the agent script generator card to define the root path for Docker agents.

### CODE REVIEW

File: web/crux-ui/src/components/nodes/dyo-node-setup.tsx [hunk 1]
```diff
@@ -33,6 +33,7 @@ const DyoNodeSetup = (props: DyoNodeSetupProps) => {
     node.install ? expiresIn(new Date(node.install.expireAt)) : null,
     () => onNodeInstallChanged(null),
   )
+  const [rootPath, setRootPath] = useState<string>('')
```
Comment (nandor-magyar) [inline web/crux-ui/src/components/nodes/dyo-node-setup.tsx:36]: It could take the default path `/srv....` - default value.

Comment (robot9706) [reply web/crux-ui/src/components/nodes/dyo-node-setup.tsx:36]: It's different on linux and mac 😅 I could also just use null.

Comment (nandor-magyar) [reply web/crux-ui/src/components/nodes/dyo-node-setup.tsx:36]: That is why it is important to be explicit about it, even a select of reasonable defaults could guide the user a bit.

File: web/crux/install-docker.sh.hbr [hunk 2]
```diff
@@ -10,10 +10,10 @@ fi
 # Setup the default root folder for volumes
 if [ $(uname -s) = 'Darwin' ]; then
   echo "Detected: Mac OS X"
-  ROOT_FOLDER=$HOME'/Library/Dyrectorio/srv/dagent'
+  ROOT_FOLDER=$HOME'{{#if rootPath}}{{{rootPath}}}{{else}}/Library/Dyrectorio/srv/dagent{{/if}}'
```
Comment (polaroi8d) [inline web/crux/install-docker.sh.hbr:13]: Remove `$HOME` too.

**Review by polaroi8d — APPROVED** [review]

**Review by nandor-magyar — APPROVED** [review]

**Pull request merged**